src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html

```diff
@@ -1,10 +1,11 @@
-<div class="row justify-content-center">
+<div class="row justify-content-center" >
```

Comment:
Is this change needed?

Reply:
This change does not affect anything, might be to the fact that we were experimenting and forgot a blank space

Reply:
Yes, I already approved this PR. What I was suggesting is that maybe we can dismiss these changes (not important tho).